### PR TITLE
add build workflow

### PR DESCRIPTION
**Pull request by nicolas-lukita**
Change log:
1. add build ci workflow

### CODE REVIEW

File: .github/workflows/integrate.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+jobs: 
+    build: 
+        runs-on: ubuntu-latest
+        steps: 
+            - uses: actions/checkout@v2
+            - uses: actions/setup-node@v2
+              with:
+                node-version: 18
+            - run: npm install
```
Comment (hanstanawi) [inline .github/workflows/integrate.yml]: Can we change to yarn? Since we use yarn instead of npm

Comment (hanstanawi) [reply .github/workflows/integrate.yml]: Reference:
https://michaelcurrin.github.io/code-cookbook/recipes/ci-cd/github-actions/workflows/node/yarn.html
@nicolas-lukita

**Comment by hanstanawi** [comment]
Will fix test error

**Comment by nicolas-lukita** [comment]
yarn test works fine if installing the dependencies using npm install. Error when installing using yarn install

**Comment by hanstanawi** [comment]
Try yarn without install. Only yarn

**Review by hanstanawi — APPROVED** [review]

**Pull request merged**